src/core/automation.py

```
@@ -13,10 +13,14 @@
 
 try:
     import Quartz
+    from Quartz import CGWindowListCopyWindowInfo, kCGWindowListExcludeDesktopElements, kCGNullWindowID
+    from Cocoa import NSScreen
```

Ruff (0.12.2)

`Cocoa.NSScreen` imported but unused; consider using `importlib.util.find_spec` to test for availability

(F401)